templates/layout.html

```diff
@@ -7,8 +7,6 @@
     <title>{{template "title"}}</title>
     <link rel="stylesheet" href="/static/css/main.css">
 </head>
-<body>
 {{template "body" .}}
```

Comment:
Everything inside the html tags here should be tabbed across and a newline should be added

Reply:
Updated 👍